### PR TITLE
Handle port and path from input

### PR DESCRIPTION
**Pull request by vzamanillo**
Related to #216

If `baseURL` has a port in the template definition then the template port should take precedence over the one from the input URL.
If the `baseURL` in the template definition has not port defined then we should use the URL port provided by input.

ej: `springboot-detect.yaml`

```
- "{{BaseURL}}/jolokia"
- "{{BaseURL}}/jolokia/list"
- "{{BaseURL}}:8090/jolokia"
- "{{BaseURL}}:8090/jolokia/list"
```
will produce

```
- "http://localhost:3000/jolokia"
- "http://localhost:3000/jolokia/list"
- "http://localhost:3000/jolokia"
- "http://localhost:3000/jolokia/list"
```
while should be

```
- "http://localhost:3000/jolokia"
- "http://localhost:3000/jolokia/list"
- "http://localhost:8090/jolokia"
- "http://localhost:8090/jolokia/list"
```

Same for URL paths, should apply only if the template definition has no path.

In short, the template definition should always take precedence.

If the user input takes precedence over the template definition then the template is not usable.

### CODE REVIEW

File: pkg/requests/bulk-http-request.go [hunk 1]
```diff
@@ -368,25 +398,32 @@ func (r *BulkHTTPRequest) parseRawRequest(request, baseURL string) (*RawRequest,
 	return &rawRequest, nil
 }
 
+// Next returns the next generator by URL
 func (r *BulkHTTPRequest) Next(reqURL string) bool {
 	return r.gsfm.Next(reqURL)
 }
+
+// Position returns the next generator by URL
```
Comment (Mzack9999) [inline pkg/requests/bulk-http-request.go:406]: I think this should be
```
// Position returns the current generator's position by URL

**Review by Mzack9999 — APPROVED** [review]
Everything looks good!

**Comment by vzamanillo** [comment]
> Everything looks good!

Thanks!!
